### PR TITLE
disemvowling

### PR DESCRIPTION
**Pull request by TrevorJamesH**

### CODE REVIEW

File: src/disemvowling.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default function disemvowling(sentence){
+  return sentence.split('').filter((character)=>!character.match(/[aeiou ]/)).join('')
+}
```
Comment (DianaVashti) [inline src/disemvowling.js:3]: This does not remove uppercase vowels. You can add them to the regex or `.toLowercase()` the string first.

File: test/disemvowling_test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { expect } from 'chai'
+import disemvowling from '../src/disemvowling'
+
+
+describe.only('disemvowling()', function() {
```
Comment (DianaVashti) [inline test/disemvowling_test.js:5]: do not include `.only` in PR

File: test/disemvowling_test.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  it('should be a function', function() {
+    expect(disemvowling).to.be.a('function')
+  })
+
+  context("When given the sentence i am groot", function() {
+    it("should return only the consinants mgrt", function() {
+      expect(disemvowling('i am groot')).to.eql('mgrt')
+    })
+  })
```
Comment (DianaVashti) [inline test/disemvowling_test.js:14]: I would make this context/it combination more concise and focused on the functionality. E.g remove the context and:
```
it('removes all spaces and lowercase vowels from a string', ...
```
and then make your second test testing for upper case letters.

File: test/disemvowling_test.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+  context("When given the sentence i dare not go where the watermelons grow", function() {
+    it("should return only the consinants mgrt", function() {
+      expect(disemvowling('i dare not go where the watermelons grow')).to.eql('drntgwhrthwtrmlnsgrw')
+    })
+  })
+})
```
Comment (DianaVashti) [inline test/disemvowling_test.js:21]: What does this test accomplish that the one above it does not? Do you expect a different functionality with longer strings? If that is the case, say so explicitly in the it('') statement

File: src/disemvowling.js (hunk 1 above)
Comment (TrevorJamesH) [reply src/disemvowling.js:3]: The issue specifically states that the input will be all lower case and with no punctuation.

https://github.com/GuildCrafts/web-development-js/issues/144

**Pull request merged**